src/spec/create_extension_spec.py

```diff
@@ -211,7 +211,7 @@ def main():
                 name="frequency",
                 doc=(
                     "The modulation frequency in Hz used for frequency domain NIRS."
```

Comment:
```suggestion
                    "The modulation frequency in Hz used for frequency domain NIRS"
```

Reply:
Good catch on the erroneous sentence @rob-luke and @rly.

Looking at the docs for neighboring attributes, I think the original text was missing an "Only used".

```suggestion
                    "The modulation frequency in Hz used for frequency domain NIRS."
                    " Only used if nirs_mode is a type of frequency domain spectroscopy."
```

Reply:
Thank you both, this makes more sense now.